game.py

```diff
@@ -55,6 +55,6 @@ def self_play_game(model,
 
     winner = env.outcome(state) if num_turns <= max_num_turns else 0
     default_v = [1, -1] * (num_turns // 2) + [1] * (num_turns % 2)
-    v = winner * default_v
+    v = [winner * value for value in default_v]
```

Comment:
Since states and action_distributions will all be numpy arrays, I think `v` should probably be one too.